### DISCUSSION TITLE
Could you add script for Umbrel

### DISCUSSION THREAD

**Discussion by blablazzz** [tteck/Proxmox#600] [Ideas]
https://umbrel.com/

**Comment by tteck** [comment]
Have you tried the CasaOS LXC?

**Reply by blablazzz** [reply]
tryed. Quite limited from the point of apps. Strange that radarr is missing from whole set of ...rr apps.

**Reply by tteck** [reply]
Done!

**Comment by blablazzz** [comment]
thanks! that was quick.
really love your work!

**Comment by fguisso** [comment]
What's the advantage of using a container system inside the proxmox? I know there are many apps in Umbrel, CasaOS, but will it be better to run these apps in lxc inside your proxmox? Maybe Umbrel needs to work with proxmox API besides docker.

This is a very real question for me because I want to try some apps in umbrel, but I'm not confident about running these apps for "production" inside a container, inside a host virtualized inside an lxc in my proxmox.
